giftcard.adoc

```diff
@@ -1,46 +1,74 @@
-## Giftcard API
+== Giftcard API
 
-### Service Description
-The Giftcard API provides extra information about purchases made through iZettle with a giftcard.
+=== Description
+====
+Giftcard API provides extra information about purchases made through iZettle with a gift card.
+====
 
-### URL
+=== URL
 https://giftcard.izettle.com
```

Comment:
If this is a base URL, I'd write Base URL. The same comment applies to similar occurrences.